10.detect-separation/scripts/nbconverted/1.separate.py

```diff
@@ -0,0 +1,324 @@
+
+# coding: utf-8
+
+# # Detect Separation Between Two Phenotypes in TCGA, GTEx, and TARGET Data
```

Comment:
Looks like you could possibly make a function to do this since you're doing something similar for the two datasets.  Though you may not have time right now and I'm not sure how specific the processing is for the two datasets.

Reply:
yeah, I could condense at least one of the pipeline steps to a function. Although the processing steps for the phenotype data is distinct

Reply:
will add in next commit!